appveyor.yml

```diff
@@ -1,4 +1,4 @@
-# Only test one combination: "Visual Studio 12 + Win64 + Debug + DLL". We can
+# Only test one combination: "Visual Studio 2015 (14) + Win64 + Debug + DLL". We can
```

Comment:
Can you help move this to line 12 (i.e., before "- language: cpp")? The comment was intended for C++ build only.